bodhi/tests/client/test_bindings.py

```diff
@@ -998,14 +998,14 @@ def test_update_as_string(self):
             text,
             "this is a string")
 
-    @mock.patch.dict(
-        client_test_data.EXAMPLE_UPDATE_MUNCH.comments[0], {u'anonymous': True})
     def test_update_with_anon_comment(self):
         """Ensure we prepend (anonymous) to a username if an anon comment is rendered"""
         client = bindings.BodhiClient()
         client.base_url = 'http://example.com/tests/'
 
-        text = client.update_str(client_test_data.EXAMPLE_UPDATE_MUNCH)
+        with mock.patch.dict(
+                client_test_data.EXAMPLE_UPDATE_MUNCH.comments[0], {u'anonymous': True}):
+            text = client.update_str(client_test_data.EXAMPLE_UPDATE_MUNCH)
```

Comment:
this sucks, I fear I'm going to have to do quite some changes in pagure if it impacts me as well :(